packages/kit-headless/playwright.config.ts

```diff
@@ -1,12 +1,34 @@
 import { workspaceRoot } from '@nx/devkit';
 import { nxE2EPreset } from '@nx/playwright/preset';
 import { defineConfig, devices } from '@playwright/test';
+import path, { dirname } from 'path';
 import { fileURLToPath } from 'url';
+import os from 'os';
 
 const __filename = fileURLToPath(import.meta.url);
 // For CI, you may want to set BASE_URL to the deployed application.
 const baseURL = process.env['BASE_URL'] || 'http://localhost:5173';
 
+const dirName = dirname(__filename);
+
+const currentPlatform = os.platform();
+
+let binaryPath;
+
+switch (currentPlatform) {
+  case 'darwin':
+    binaryPath = './browsers/chrome/113/chrome-darwin/Chromium.app';
```

Comment:
Need someone with a Mac to test this.

Reply:
@thejackshelton you have a mac now right? :)
did you have a chance to check this out?

Reply:
Just checked this out. I got a downloading screen that went to 100% -> ran the tests, and running any test immediately fails.

@cwoolum does this mean the binary path is incorrect?